fe1-web/src/features/social/reducer/SocialReducer.ts

```diff
@@ -277,15 +277,15 @@ export const makeChirpsListOfUser = (laoId?: Hash) => (user?: PublicKey) => {
 };
 
 const createReactionsEntry = (reactionByUser: Record<string, string[]>) => ({
-  '👍': reactionByUser['👍'] ? reactionByUser['👍'].length : 0,
-  '👎': reactionByUser['👎'] ? reactionByUser['👎'].length : 0,
-  '❤️': reactionByUser['❤️'] ? reactionByUser['❤️'].length : 0,
+  '👍': reactionByUser['👍'] ? reactionByUser['👍'].map(PublicKey.fromState) : [],
```

Comment:
isn't this format (and its usage) going to severely degrade the runtime characteristics (RAM usage, responsiveness) of the application ?

Reply:
Most likely, yes.. Not doing the `.map(PublicKey.fromState)` would already be better performance-wise but it contradicts the pattern we want to use in general.

A most likely better solution would be to revert the changes to this selector and add a new one that is just there to check whether the user already reacted :)